Reconstruction/doc/RecCalorimeter.md

```diff
@@ -15,22 +15,19 @@ ECAL calorimeter description in `Detector/DetFCChhECalSimple`:
 
 Digitisation creates cells out of simulated energy deposits. From the EDM point of view, both input and output of the digitisation uses `fcc::CaloHit`.
 The input (simulated deposits) contains raw information about the energy deposited in the cells of the sensitive volumes.
-The ouput (cells) contains calibrated energy and the noise. It may also use different segmentation than the original cells of the sensitive volumes.
+The output (cells) contains energy (corrected for the losses in the passive layers) and the noise. The cells may correspond to the active volumes or to the segmentation cells. In particular, different segmentation may be used than the original cells of the sensitive volumes used in the simulation.
 
-> Note: some resegmentation procedures (`RedoSegmentation`) expect the truth position of the energy deposits, hence they require `fcc::PositionedCaloHit` collection.
+> Note: resegmentation procedures (`RedoSegmentation`) expect the truth position of the energy deposits, hence they require `fcc::PositionedCaloHit` collection.
 
-`CreateCaloCells` is an algorithm for the digitisation. It calls 3 tools to perform the deposits merging, energy calibration and noise addition.
+`CreateCaloCells` is an algorithm for the digitisation. It calls tools to perform the energy calibration and cell noise addition. It is geometry agnostic and can be used to merge any calorimeter hit collection (unless the noise is added, [see 3.](3-noise-addition)).
 
-## 1. Deposits merging
+## 1. Energy deposits merging
 
 The output of the simulation contains a large collection of energy deposits. First step towards the digitisation is merging those hits into the hit collection with one hit per cell of the sensitive volume. The energy of such hit is a sum of all the deposits within that sensitive volume cell.
+`CreateCaloCells` does not take time into account. The current implementation allows introduction of a time threshold (i.a. energy deposits with timing above threshold would be neglected). For any more complicated digitisation, new digitisation algorithm should be developed.
```

Comment:
typo: i.a. -> i.e.

I find this a bit confusing: It does not take time into account but you allow to introduce a time threshold, isn't that a contradiction?